src/utils.js

```diff
@@ -1,6 +1,5 @@
-import jQuery from 'jquery';
-import testJQuery from './utils/jQueryMock';
 import { default as config } from './config.js';
+import jquery from './utils/jQueryModule';
```

Comment:
I would name it just `jQuery` instead of `jQueryModule` - every single file is actually a module so it's kinda redundant.